server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/DBTableController.java

```diff
@@ -122,4 +123,15 @@ public ListResponse<DataType> getPartitionKeyDataTypes(@PathVariable String sess
         return Responses.list(this.partitionPlanService.getPartitionKeyDataTypes(sessionId, databaseId, tableName));
     }
 
+    @PutMapping(value = "/{sessionId}/databases/{databaseName}/externalTables/"
+            + "{externalTableName}/syncExternalTableFiles")
+    public SuccessResponse syncExternalTableFiles(@PathVariable String sessionId,
+            @PathVariable(required = false) String databaseName,
+            @PathVariable(required = true, name = "externalTableName") String externalTableName) {
+        Base64.Decoder decoder = Base64.getDecoder();
+        externalTableName = new String(decoder.decode(externalTableName));
+        ConnectionSession session = sessionService.nullSafeGet(sessionId, true);
+        tableService.syncExternalTableFiles(session, databaseName, externalTableName);
+        return Responses.empty();
```

Comment:
Why return `empty`?

Reply:
empty response indicates that synchronization is successful

Reply:
use Boolean